tools/publish_helm_gcs.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+HELM_TARGET_DIR=$(pwd)/tmp/incubator
+readonly HELM_URL=https://storage.googleapis.com/kubernetes-helm
+readonly HELM_TARBALL=helm-v2.9.1-linux-amd64.tar.gz
+#readonly STABLE_REPO_URL=https://orange-kubernetes-charts.storage.googleapis.com/
+readonly INCUBATOR_REPO_URL=https://orange-kubernetes-charts-incubator.storage.googleapis.com/
+#readonly GCS_BUCKET_STABLE=gs://orange-kubernetes-charts
+readonly GCS_BUCKET_INCUBATOR=gs://orange-kubernetes-charts-incubator
+
+main() {
+    mkdir -p tmp
+    setup_helm_client
+    authenticate
+
+#    if ! sync_repo stable "$GCS_BUCKET_STABLE" "$STABLE_REPO_URL"; then
+#        log_error "Not all stable charts could be packaged and synced!"
+#    fi
```

Comment:
Any reason to still keep this ^ ?

Reply:
To uncomment it, once we will have a stable versions.